core/templates/components/common-layout-directives/navigation-bars/top-navigation-bar.component.html

```diff
@@ -215,9 +214,8 @@
               (mouseover)="openSubmenu($event, 'aboutMenu')"
               (mouseleave)="closeSubmenuIfNotMobile($event)">
             <li class="nav-item">
-              <a id="navbar-about-menu-about-button"
-                 (keydown)="onMenuKeypress($event, 'aboutMenu', {shiftTab: ACTION_CLOSE, enter: ACTION_CLOSE})"
-                 class="dropdown-item nav-link oppia-navbar-tab-content e2e-test-about-link about-link"
+              <a (keydown)="onMenuKeypress($event, 'aboutMenu', {shiftTab: ACTION_CLOSE, enter: ACTION_CLOSE})"
+                 class="dropdown-item nav-link oppia-navbar-tab-content e2e-test-about-link about-link e2e-test-navbar-about-menu-about-button"
```

Comment:
I think you only need one e2e-test class per HTML element, right? You can reuse that class in multiple tests. Ditto below.